pkg/monitor/monitor/template_test.go

```diff
@@ -517,10 +535,8 @@ scrape_configs:
     action: replace
 `
 	model := &MonitorConfigModel{
-		ReleaseTargetRegex: &target,
-		ReleaseNamespaces: []string{
-			"ns1",
-			"ns2",
+		ClusterInfos: []ClusterRegexInfo{
+			{Name: "target", Namespace: "ns1"},
```

Comment:
It's better to verify multi tidbcluster here.

Reply:
ok,i take time to make up.